packages/bot-skeleton/src/scratch/hooks/tooltip.js

```diff
@@ -0,0 +1,70 @@
+import { blueInfo } from '../blocks/images';
+
+Blockly.Tooltip.HOVER_MS = 50;
+
+Blockly.Tooltip.show_ = () => {
+    let params;
+    Blockly.Tooltip.DIV = ((Blockly.Tooltip.poisonedElement_ = Blockly.Tooltip.element_), Blockly.Tooltip.DIV);
+    if (!Blockly.Tooltip.blocked_) {
+        Blockly.Tooltip.DIV.innerHTML = '';
+        for (params = Blockly.Tooltip.element_.tooltip; typeof params === 'function'; ) params = params();
+        params = Blockly.utils.string.wrap(params, Blockly.Tooltip.LIMIT);
+        params = params.split('\n');
+
+        params.forEach((param, index) => {
+            const div = document.createElement('div');
+            const text_span = document.createElement('span');
+
+            text_span.appendChild(document.createTextNode(param));
+            text_span.style.verticalAlign = 'middle';
+            Blockly.Tooltip.DIV.appendChild(div);
+
+            if (index === 0) {
```

Comment:
Also a suggestion, how about `!index`?

```suggestion
            if (!index) {
```